src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageReadmeControl.xaml.cs

```diff
@@ -73,21 +144,26 @@ private void UserControl_DataContextChanged(object sender, DependencyPropertyCha
             }
         }
 
-        private void PackageReadmeControl_Unloaded(object sender, RoutedEventArgs e)
+        private void PackageReadmeControl_Loaded(object sender, RoutedEventArgs e)
         {
-            if (ReadmeViewModel is not null)
+            NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async () =>
```

Comment:
Another fire and forget that adds JTF overhead for no reason, see: https://github.com/NuGet/NuGet.Client/pull/6190#discussion_r1889485483.